From 9866475005e7b17adf15d13d0584273086d32c7c Mon Sep 17 00:00:00 2001
From: Benjamin Drung <benjamin.drung@profitbricks.com>
Date: Thu, 27 Jul 2017 19:09:24 +0200
Subject: [PATCH] Fix typo availible -> available

Signed-off-by: Benjamin Drung <benjamin.drung@profitbricks.com>
---
 src/perftest_parameters.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/perftest_parameters.c b/src/perftest_parameters.c
index 92c2570..da41b37 100755
--- a/src/perftest_parameters.c
+++ b/src/perftest_parameters.c
@@ -327,7 +327,7 @@ static void usage(const char *argv0, VerbType verb, TestType tst, int connection
 		}
 
 		printf("  -T, --tos=<tos value> ");
-		printf(" Set <tos_value> to RDMA-CM QPs. availible only with -R flag. values 0-256 (default off)\n");
+		printf(" Set <tos_value> to RDMA-CM QPs. available only with -R flag. values 0-256 (default off)\n");
 	}
 
 	printf("  -u, --qp-timeout=<timeout> ");
@@ -2008,7 +2008,7 @@ int parser(struct perftest_parameters *user_param,char *argv[], int argc)
 				  } break;
 			case 'C':
 				  if (user_param->tst != LAT) {
-					  fprintf(stderr," Availible only on Latency tests\n");
+					  fprintf(stderr," Available only on Latency tests\n");
 					  return 1;
 				  }
 				  user_param->r_flag->cycles = ON;
-- 
2.14.1
